client/src/components/Drug/DrugRecord/DrugRecord.tsx

```diff
@@ -163,41 +165,29 @@ export const DrugRecord: React.FC = () => {
       ),
     },
     {
-      name: "Categories",
+      name: "FDA Applications",
       sectionContent: (
         <Box className="box-content">
           <Table>
             <TableBody>
-              {drugData?.geneCategories ? drugData?.geneCategories?.map((category: any) => {
+              {drugData?.drugApplications ? drugData?.drugApplications?.map((app: any, i: number) => {
+                const appId = app.appNo.match(/\d+/)
+                const url = `https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm?event=overview.process&ApplNo=${appId}`
                 return (
-                  <TableRow key={category.name}>
-                    <TableCell className="attribute-name">{category.name}:</TableCell>
+                  <TableRow key={i}>
+                    <TableCell className="attribute-name">
+                      {
+                        <Link target="_blank" href={url}>{app.appNo} <LaunchIcon sx={{ fontSize: 13 }}/></Link>
+                      }
+                    </TableCell>
                   </TableRow>
                 )
-              }) : noData}
-            </TableBody>
-          </Table>
-        </Box>
-      ),
-    },
-    {
-      name: "Publications",
-      sectionContent: (
-        <Box className="box-content publication-item">
-          <Table>
-            <TableBody>
-              {drugData?.geneClaims ? drugData?.geneClaims?.map((claim: any) => {
-                return (
-                  <TableRow key={claim?.source?.citation}>
-                    <TableCell className="attribute-name">{claim?.source?.citation}:</TableCell>
-                  </TableRow>
-                )
-              }) : noData}
+                }) : noData}
```

Comment:
if no drugApplications exist for a record, the box renders but the No Data filler text does not (example record, gilteritinib). I think adding .length to the first {drugData?.drugApplications ? fixes this